ThemeDecorator/ThemeDecorator.js

```diff
@@ -34,6 +34,7 @@ const defaultConfig = /** @lends agate/ThemeDecorator.ThemeDecorator.defaultConf
 	 *
 	 * @type {Boolean}
 	 * @default true
+	 * @memberof agate/ThemeDecorator.ThemeDecorator.defaultConfig
```

Comment:
Could you explain why `memberof` is needed for these props?

Reply:
That's because `@memberof` is  one of tags which should be documented for defaultConfig.